cypress/cypress.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     openMode: 1,
   },
   e2e: {
+    testIsolation: true,
```

Comment:
[Setting to have Cypress clear all relevant state between tests](https://docs.cypress.io/guides/core-concepts/test-isolation). Should probably be on given the effort we put into making the tests rerunnable